cuda/test/solver/bicgstab_kernels.cpp

```diff
@@ -261,7 +261,7 @@ TEST_F(Bicgstab, CudaBicgstabInitializeIsEquivalentToRef)
     GKO_EXPECT_MTX_NEAR(d_beta, beta, 1e-14);
     GKO_EXPECT_MTX_NEAR(d_gamma, gamma, 1e-14);
     GKO_EXPECT_MTX_NEAR(d_omega, omega, 1e-14);
-    GKO_ASSERT_ARRAY_EQ(d_stop_status, stop_status);
+    GKO_ASSERT_ARRAY_EQ(*d_stop_status, *stop_status);
```

Comment:
Line 151 above. `d_stop_status` is being created twice ? Maybe in other files as well.

Reply:
It's being constructed once and assigned once (hence the *), so that should be fine

Reply:
If you look line 151, there really are two times `d_stop_status = std::unique_ptr<...`.

Reply:
I see, thanks! I was looking in the wrong file